zerver/lib/home.py

```diff
@@ -200,6 +204,12 @@ def build_page_params_for_home_page_load(
         furthest_read_time=furthest_read_time,
         bot_types=get_bot_types(user_profile),
         two_fa_enabled=two_fa_enabled,
+        apps_page_url="/apps/" if settings.ZILENCER_ENABLED else "https://zulip.com/apps/",
```

Comment:
Let's extract this as a function that `zerver/context_processors.py` exports for us to call.